tests/queries/tests.py

```diff
@@ -1164,6 +1164,47 @@ def test_ticket_20250(self):
             ['<Author: a1>', '<Author: a2>', '<Author: a3>', '<Author: a4>']
         )
 
+    def test_ticket_16891(self):
```

Comment:
a semantic test name is preferred when it's easy to describe the feature being tested (e.g. `test_delete_returns_num_rows`)